juniper_official/oam/ethernet-oam-lfm.rule

```diff
@@ -27,20 +27,34 @@ healthbot {
                 }
                 type string;
                 description "Discovery state for a given interface";
+                enumerate "Passive Wait" as 0;
+                enumerate "Send Any" as 1;
```

Comment:
I just spoke to Venkat and as per him 2 and 3 are not good states. So it has to be less than 1